10-block/2/exercise4_2.js

```diff
@@ -0,0 +1,34 @@
+const messageDelay = () => Math.floor(Math.random() * 5000);
+
+const getMarsTemperature = () => {
+  const maxTemperature = 58;
+  return Math.floor(Math.random() * maxTemperature);
+};
+
+const toFahrenheit = degreeCelsius => (degreeCelsius * 9) / 5 + 32;
+const temperatureInFahrenheit = temperature =>
+  console.log(`It is currently ${toFahrenheit(temperature)}ºF at Mars`);
+const greet = temperature =>
+  console.log(
+    `Hi there! Curiosity here. Right now is ${temperature}ºC at Mars`
+  );
+
+const handleError = errorReason =>
+  console.log(`Error getting temperature: ${errorReason}`);
+
+const isRobotBusy = () => (Math.floor(Math.random() * 101) < 60 ? false : true);
```

Comment:
você não precisa usar o floor da pra comparar o resultado de random * 101 < 60 da mesma forma,
```suggestion
const isRobotBusy = () => ((Math.random() * 101) < 60 ? false : true);
``` 
e também não precisa multiplicar por 101
```suggestion
const isRobotBusy = () => (Math.random() < 0.6 ? false : true);
```

Reply:
Eu também gostaria de acrescentar só pra deixar mais curtinho. Não precisa usar o ternário também. Se fizer Math.random() < 0.6 ele já vai retornar true ou false.

Reply:
Usando um boolean para retornar um boolean hahahah realmente, não faz sentido!
Boa Rangel, valeu!